dali/pipeline/operator/op_schema.h

```diff
@@ -90,6 +92,27 @@ class DLL_PUBLIC OpSchema {
     return *this;
   }
 
+  /**
+   * @brief Sets the docstring for input.
+   *
+   * The inputs should be registered in the order they will appear in the signature.
+   * TODO(klecki): consider registering them for given position
+   * If the operator specifies some range of allowed inputs with NumInput(int min, int max)
+   * only the first `min` inputs are considered mandatory, the rest are optional
+   *
+   * Will generate entry in `Args` section:
+   *     name (type_doc): doc
+   *
+   * @param name Name of the input, it will also appear in the doc for `__call__` signature
+   * @param type_doc
+   * @param doc
+   */
+  DLL_PUBLIC inline OpSchema &InputDocStr(const string &name, const string &type_doc,
```

Comment:
I don't like this API. Either use `AddInput` / `AddOptionalInput`, analogous to `AddArg` / `AddOptionalArg` or make input index a parameter of this function (`InputDocStr(int index, const string &name, ...)`)

Reply:
We decided on indexed docstrings for inputs.